src/cascadia/TerminalSettingsEditor/Profiles.h

```diff
@@ -49,6 +51,9 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         void UseParentProcessDirectory(const bool useParent);
         bool UseCustomStartingDirectory();
 
+        // tab display name
```

Comment:
IMO, this comment isn't adding anything lol.
```suggestion
```

Reply:
Ah this reminded me that the entire tab display name stuff needs to be removed, thanks!